src/errors.ts

```diff
@@ -115,6 +115,7 @@ export interface IInteractiveError {
 
 export module InteractiveError {
     export class Base extends BaseError {
+        public path: string;
```

Comment:
Readonly doesn't work here, as we got to assign the path in our workaround in line 132.
The reason for this is to prevent a change to the constructor parameters that would break stuff.